doc/nrf/gs_installing.rst

```diff
@@ -53,6 +52,12 @@ The installation process is different depending on your operating system.
          :end-before: #. Close the window and open a new
 
       Ensure that these dependencies are installed with their versions as specified in the :ref:`Required tools table <req_tools_table>`.
+      To check the list of installed packages and their versions run the following command:
```

Comment:
Just a doubt - do we have this command for linux and Mac OS ?
```suggestion
      To check the list of installed packages and their versions, run the following command:
```

Reply:
We are checking the versions of Linux OS in Docs. 
![image](https://user-images.githubusercontent.com/91891495/202995502-08a1892f-9462-4f71-bba2-cdc3099ef6f0.png)

For Mac OS - I found this command useful - "brew list --versions". 
We could add this?

Reply:
This is great, thank you Divya!

Reply:
updated